src/main/java/com/google/sps/data/CareerQuestionDatabase.java

```diff
@@ -0,0 +1,36 @@
+package com.google.sps.data;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.FetchOptions;
+import java.util.ArrayList;
+
+/* purpose: to manage the interface of the Career Question database */
+public final class CareerQuestionDatabase {
+ 
+  private DatastoreService datastore;
+  private ArrayList<CareerQuestionAndResponses> questionAndResponses;
+  
+  public CareerQuestionDatabase(DatastoreService datastore) {
+    this.datastore = datastore;
+  }
+
+  /* method to return all questions and responses from database as an arraylist */
+  public ArrayList<CareerQuestionAndResponses> getQuestionsAndResponses() {
+  // TODO: replace hard coded example Q and Reponse with entities queried from datastore
+    CareerResponse HARD_CODED_RESPONSE1 = new CareerResponse("response1", "career1");
+    CareerResponse HARD_CODED_RESPONSE2 = new CareerResponse("response2", "career2");
```

Comment:
String to variable

Reply:
Since this is just a hardcoded dummy response, I decided not to represent the string with a private final variable.